plugins/arcgis/web-app/projects/main/src/lib/arc-admin/arc-admin.component.ts

```diff
@@ -69,14 +72,79 @@ export class ArcAdminComponent implements OnInit {
     this.config = config;
     this.configChangedNotifier.next();
   }
-
+// Visually blank but has underlying value in editConfig
   ngOnInit(): void {
-  }
+    this.attributesForm = this.fb.group({
+      observationIdField: [''], 
+      idSeparator: [''],
+      eventIdField: [''],
+      lastEditedDateField: [''],
+      eventNameField: [''],
+      userIdField: [''],
+      usernameField: [''],
+      userDisplayNameField: [''],
+      deviceIdField: [''],
+      createdAtField: [''],
+      lastModifiedField: [''],
+      geometryType: ['']
+    });
+  }
+  
+  //Save attributes form. If left blank, assign default value
```

Comment:
What will the default values be? From server config?

Reply:
Yep! From the ArcGISPluginConfig file